Javascript/jq-test.js

```diff
@@ -1,69 +1,17 @@
-$( document ).ready(function() {
-    console.log( "ready!" );
-
+$(function() {
     $('#settings').click(function(){
       $('.bkg-menu').removeClass("hide");
+      $('#set-title').addClass("hide");
     });
 
     $('#profile').click(function(){
       $('.bkg-menu').removeClass("hide");
+      $('#pro-title').addClass("hide");
     });
 
     $('#menu-close').click(function(){
       $('.bkg-menu').addClass("hide");
-    });
-
-    $(function() {
-
-        $('#settings').click(function() {
-
-            if ($('#set-title').hasClass("hide")) {
-                $('#set-title').removeClass('hide');
-            }
-            else if (!$('#set-title').hasClass("hide")) {
-                $('#set-title').addClass('hide');
-            }
-
-        });
-
-    });
-
-    $(function() {
-
-        $('#profile').click(function() {
-
-            if ($('#pro-title').hasClass("hide")) {
-                $('#pro-title').removeClass('hide');
-            }
-            else if (!$('#pro-title').hasClass("hide")) {
-                $('#pro-title').addClass('hide');
-            }
-
-        });
-
-    });
-
-    $(function() {
-
-        $('#menu-close').click(function() {
-
-            if (!$('#set-title').hasClass("hide")) {
-                $('#set-title').addClass('hide');
-            }
-
-        });
-
-    });
-
-    $(function() {
-
-        $('#menu-close').click(function() {
-
-            if (!$('#pro-title').hasClass("hide")) {
-                $('#pro-title').addClass('hide');
-            }
-
-        });
-
+      $('#set-title').addClass('hide');
+      $('#pro-title').addClass('hide');
```

Comment:
Not added an if statement here as if `#pro-title` already has the class `hide`, `$('#pro-title').addClass('hide');` will do nothing anyway